features/container-queries.yml.dist

```diff
@@ -2,8 +2,9 @@
 # Do not edit this file by hand. Edit the source file instead!
 
 status:
-  baseline: low
+  baseline: high
   baseline_low_date: 2023-02-14
+  baseline_high_date: 2025-08-14
```

Comment:
I _think_ this is safe, but the data in BCD is not complete. See https://github.com/mdn/browser-compat-data/issues/25766 (and relatedly, https://github.com/mdn/browser-compat-data/issues/25766).